client/db.go

```diff
@@ -158,6 +158,9 @@ func Open(addr string) (*DB, error) {
 	if err != nil {
 		return nil, err
 	}
+	if u.User == nil {
+		u.User = url.User("root")
```

Comment:
In the long run the user name will be embedded in the certificate, so you won't need to specify it in the url at all (or we'd use the "user" field of the url to choose which certificate to use). In the meantime "root" is probably a more sensible default than `$USER`.